app/scripts/lib/router.js

```diff
@@ -129,6 +129,7 @@ function (
       'confirm_reset_password(/)': showView(ConfirmResetPasswordView),
       'cookies_disabled(/)': showView(CookiesDisabledView),
       'force_auth(/)': showView(ForceAuthView),
+      'force_auth_complete(/)': showView(ReadyView, { type: 'force_auth' }),
```

Comment:
Ok we should open an issue if needed for throwing on no-session access to http://localhost:3030/force_auth_complete